helm/charts/infra/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: infra
+description: A Helm chart for common shared infrastructure resources.
+
+dependencies:
+- name: base
+  version: "0.1.0"
+  repository: "file://../base"
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
Let's remove the default helm chart comments

Reply:
Ok